content/efcore/getting-started/README.md

```diff
@@ -0,0 +1,11 @@
+# Getting Started with Entity Framework Core
+
+## Who Should Read This?
+
+This tutorial is meant for those new to Entity Framework (EF) Core, as it covers the basic functionalities of EF Core. More advanced functionality is covered in the [docs](https://docs.efproject.net/en/latest/) . Before beginning this tutorial, you should understand a few topics. 1) this tutorial assumes a familiarity with C#. If you're new to C#, you may wish to go through the [Getting Started with C#](https://www.microsoft.com/net/tutorials/csharp/getting-started) tutorial. 2) you should have a basic knowledge of relational database concepts. 3) this tutorial will use LINQ. If you have not used LINQ before, the [Introducing LINQ](https://www.microsoft.com/net/tutorials/csharp/getting-started/linq) tutorial can help you get up-to-speed on LINQ.
```

Comment:
Could you remove "on LINQ" from the last sentence? Just "can help you get up-to-speed."